volume-providers/vpc/messages/messages_en.go

```diff
@@ -51,6 +59,27 @@ var messagesEn = map[string]util.Message{
 		RC:          400,
 		Action:      "Please check the snapshot ID once, You many need to verify by using 'ibmcloud is' cli.",
 	},
+	string(VolumeAttachFindFailed): util.Message{
+		Code:        string(VolumeAttachFindFailed),
+		Description: "Failed to find the volume attachment by using volume ID :'%s' and instance ID : '%s'.",
+		Type:        "RetrivalFailed",
+		RC:          400,
+		Action:      "Please check the volume attachment once, You many need to verify by using 'ibmcloud is in-vol' cli.",
+	},
+	string(VolumeAttachFailed): util.Message{
+		Code:        string(VolumeAttachFailed),
+		Description: "Failed to attach volume :'%s' to  instance : '%s'.",
+		Type:        "AttachDetachFailed",
```

Comment:
ambiguity for error Type, can we have either AttachFailed or DetachFailed

Reply:
Will change